2024.html

```diff
@@ -62,6 +62,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://boehs.org/">Evan Boehs</a></li>
             <li><a href="https://weblog.200ok.com.au/">Ben Buchanan</a></li>
             <li><a href="https://chrisburnell.com/">Chris Burnell</a></li>
+            <li><a href="https://charakterziffer.github.io/">charakterziffer</a></li>
```

Comment:
Please move one up (sort by alphabet).

(Your call, but please also check on case and if you want to disclose your name, in case of a personal website.)

Reply:
@j9t: Thanks for al the work.

Since we’re sorting by alphabetically order, “charakterziffer” comes after “Burnell”, so I’ll merge this. We can still change things if need be.

Reply:
Strange, not sure what happened here. Must have missed something.

Also not sure about the note to sort alphabetically, @arkhi, if that’s precisely what I asked to do.